microsoft/testsuites/core/timesync.py

```diff
@@ -129,22 +131,33 @@ def timesync_validate_ptp(self, node: Node) -> None:
     def timesync_check_unbind_clocksource(self, node: Node) -> None:
         # 1. Check clock source name is one of hyperv_clocksource_tsc_page,
         #  lis_hv_clocksource_tsc_page, hyperv_clocksource.
-        clocksource = [
+        clocksource_x86 = [
```

Comment:
Because we have str of architecture already, so the logic can be simpler by use dict, like below

```python
clocksource_map = {
  Architecture.X64:  
       ["hyperv_clocksource_tsc_page",
            "lis_hyperv_clocksource_tsc_page",
            "hyperv_clocksource",
            "tsc",
        ],
  Architecture.ARM64:  [
            "arch_sys_counter",
        ]}
```

```python
    clocksource = clocksource_map.get(arch, None)
    if not arch:
        raise UnsupportedArchException(arch)
```

Reply:
Done